### ISSUE TITLE
[Enhancement] Option to disable block breaking particles

### ISSUE THREAD

**Issue by ZenIsBestWolf** [sp614x/optifine#7072]
## Disable block breaking particles

Received via email, some users have large farms that require breaking a lot of blocks automatically (i.e. cactus farm) and the option to disable block breaking particles would be a major quality of life improvement for these users, as well as adding further optimizations for users with poor hardware.

**Comment by mindforger** [comment]
![grafik](https://user-images.githubusercontent.com/2683555/185409790-5bba1605-daa8-4e2b-b76f-5a7c8431eed3.png)
 isn't this already an option?

**Comment by fayer3** [comment]
no, I checked it. setting particles to minimal, still produces breaking particles

**Comment by mindforger** [comment]
ah okay i did actually not test it :P but thought this should also disable them
